packages/db/src/schema/roleRequest.ts

```diff
@@ -0,0 +1,48 @@
+import { relations, sql } from "drizzle-orm";
+import { pgTable, text, timestamp, uuid, varchar } from "drizzle-orm/pg-core";
+import { createInsertSchema } from "drizzle-zod";
+import { z } from "zod";
+
+import { Company } from "./companies";
+import { RequestStatus } from "./misc";
+import { Profile } from "./profiles";
+
+export const RoleRequest = pgTable("role_request", {
+  id: uuid("id").notNull().primaryKey().defaultRandom(),
+  roleTitle: varchar("title").notNull(),
+  roleDescription: text("description"),
+  companyId: varchar("companyId").notNull(),
+  createdAt: timestamp("createdAt").defaultNow().notNull(),
+  status: varchar("status", {
+    enum: ["PENDING", "APPROVED", "REJECTED"], // Explicitly list the enum values (?)
```

Comment:
I think this is good, if anything it's even more type robust than the "workEnviornment" one we use in reviews rn!